appium/webdriver/extensions/location.py

```diff
@@ -80,7 +80,7 @@ def location(self) -> Dict[str, float]:
                 - longitude (float)
                 - altitude (float)
         """
-        return self.execute(Command.GET_LOCATION).get('value', {})
+        return self.execute(Command.GET_LOCATION)['value']  # pylint: disable=unsubscriptable-object
```

Comment:
this might be an issue of pylint itself, because self.execute is known to return Dict
🤷